darwin/dataset/remote_dataset.py

```diff
@@ -198,7 +203,8 @@ def pull(
         use_folders: bool = False,
         video_frames: bool = False,
     ):
-        """Downloads a remote project (images and annotations) in the datasets directory.
+        """
+        Downloads a remote project (images and annotations) in the datasets directory.
```

Comment:
I don't see project used often; perhaps another find replace for project -> dataset?
But then this comment would sound odd; so probably needs to change to something like

        Downloads a remote dataset (images and annotations) to local working directory.

Reply:
Didn't exactly use your wording, but I agree on the project part. Fixed.